src/sdk-reference/cpp/1/offline-tools/index.md

```diff
@@ -7,69 +7,74 @@ order: 100
 
 # Offline tools
 
-The Kuzzle SDK provides a set of properties that help your application to be resilient to the loss of network connection
-during its lifespan. 
+The Kuzzle SDK provides a set of options and methods that helps your application to be resilient to the loss of network connection
```

Comment:
Since @Aschen is off for 3 weeks, I'm assigning this PR to myself.

About your suggestion: I would agree if the sentence was "... provides options and methods...", but here isn't that "the set" that provides help?

Or in doubt, we also could write "...a set of options and methods helping your application..."

What do you think?

Reply:
I agree with @scottinet, here it's "the set" that provides help